packages/core/src/mixins/__tests__/rowComp.test.js

```diff
@@ -60,41 +64,40 @@ it('takes a React Element as icon', () => {
 });
 
 it('renders <Text> with adjusted alignment', () => {
-    const leftWrapper = shallow(<RowCompFoo align="left" basic="Basic" />);
-    const centerWrapper = shallow(<RowCompFoo align="center" basic="Basic" />);
-    const centerIconWrapper = shallow(<RowCompFoo align="center" basic="Basic" icon="add" />);
-    const rightWrapper = shallow(<RowCompFoo align="right" basic="Basic" />);
-    const reverseWrapper = shallow(<RowCompFoo align="reverse" basic="Basic" />);
-
-    expect(
-        leftWrapper
-            .find(Foo).shallow()
-            .find(Text).prop('align')
-    ).toBe('left');
-
-    expect(
-        centerWrapper
-            .find(Foo).shallow()
-            .find(Text).prop('align')
-    ).toBe('center');
-
-    expect(
-        centerIconWrapper
-            .find(Foo).shallow()
-            .find(Text).prop('align')
-    ).toBe('left');
-
-    expect(
-        rightWrapper
-            .find(Foo).shallow()
-            .find(Text).prop('align')
-    ).toBe('right');
-
-    expect(
-        reverseWrapper
-            .find(Foo).shallow()
-            .find(Text).prop('align')
-    ).toBe('right');
+    // Left-aligned
```

Comment:
Looks like each comment like this could be an `it(...)`, XD
I think generally that would be better since you can find failing test easier.

Reply:
yeah sounds great to me